fuse_constraints/test/test_absolute_constraint.cpp

```diff
@@ -246,8 +278,8 @@ TEST(AbsoluteConstraint, Optimization)
     mean1 << 1.0, 2.0, 3.0;
     fuse_core::Matrix3d cov1;
     cov1 << 1.0, 0.0, 0.0,
-            0.0, 1.0, 0.0,
-            0.0, 0.0, 1.0;
+      0.0, 1.0, 0.0,
```

Comment:
nit, `INDENT-OFF` here too

Reply:
https://github.com/locusrobotics/fuse/pull/298/commits/73be208b78ca87589e283c66cb50df27dd3d33b0 !

I also fixed the copyright (I had to make extremely minor changes to get ament_copyright to be satisfied), and fixed the pragma warning on fuse_core/macros.h

(I realized what happened was macros.h got migrated macros.hpp (when it was an already deprecated file, so I pointed both of them to fuse_macros.hpp.)

I can also remove macros.hpp if that's preferable (since macros.h is supposed to be deprecated anyway?)